spring-cloud-netflix-ribbon/src/main/java/org/springframework/cloud/netflix/ribbon/RibbonProperties.java

```diff
@@ -22,7 +22,11 @@
 import static com.netflix.client.config.DefaultClientConfigImpl.DEFAULT_READ_TIMEOUT;
 
 public class RibbonProperties {
-    private final IClientConfig config;
+	private final IClientConfig config;
```

Comment:
indentation is off

Reply:
Fixed. RibbonProperties use different indentation than other files.